DOC-2567 DB Console proxy to any node (#14435)
cockroachdb/docs: v21.2/ui-overview.md

For guidance on accessing the DB Console in the context of cluster deployment, see [Start a Local Cluster](start-a-local-cluster.html) and [Manual Deployment](manual-deployment.html).

### Proxy DB Console

> If your CockroachDB cluster is behind a load balancer, you may wish to proxy your DB Console connection to a different node in the cluster from the node you first connect to. You can accomplish this using one of these methods:

Reviewer: ```suggestion
If your CockroachDB cluster is behind a load balancer, you may wish to proxy your DB Console connection to a different node in the cluster from the node you first connect to. This is useful in deployments where a third-party load balancer otherwise determines which CockroachDB node you connect to in DB Console, or where web management access is limited to a subset of CockroachDB instances in a cluster.

You can accomplish this using one of these methods:
```

Reviewer 2: Just to note @andf-crl I assume Michael's suggestions would apply to v22.1 as well.

> - Once connected to DB Console, use the **Web server** dropdown menu from the [Advanced Debug page](ui-debug-pages.html#license-and-node-information) to select a different node to proxy to.

Reviewer 2: ```suggestion
- Once connected to DB Console, use the **Web server** dropdown menu from the [**Advanced Debug** page](ui-debug-pages.html#license-and-node-information) to select a different node to proxy to.
```

> - Use the `remote_node_id` parameter in your DB Console URL to proxy directly to a specific node, e.g., use `http://<host>:<http-port>/?remote_node_id=2` to proxy directly to node `2`. 

Reviewer 2: ```suggestion
- Use the `remote_node_id` parameter in your DB Console URL to proxy directly to a specific node. For example, use `http://<host>:<http-port>/?remote_node_id=2` to proxy directly to node `2`. 
```

> This is useful in deployments where a third-party load balancer otherwise determines which CockroachDB node you connect to in DB Console, or where web management access is limited to a subset of CockroachDB instances in a cluster.

Reviewer: Moved this up to the first paragraph in the section
```suggestion
```
